### PR TITLE
remove wrench topics from toc

### PR DESCRIPTION
**Pull request by tdykstra**

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -1,6 +1,41 @@
 { 
   "redirections": [ 
     {   
+        "source_path": "aspnetcore/client-side/cdn.md", 
+        "redirect_url": "/aspnet/core//client-side/",
```
Comment (GuardRex) [inline .openpublishing.redirection.json]: Should this be ...

/aspnetcore/client-side/

Comment (GuardRex) [inline .openpublishing.redirection.json]: @tdykstra I guess ur moving stuff around; however, I wonder about the double-slash here ... intentional?

Comment (tdykstra) [inline .openpublishing.redirection.json]: @GuardRex No, typo, thanks for catching it.

Comment (GuardRex) [inline .openpublishing.redirection.json]: I'm tasked with doing the same over here and using your PR as a guide ... a nice, fun little Monday morning PR. :grinning:

**Pull request merged**